Assets/MixedRealityToolkit.Services/SpatialAwarenessSystem/MixedRealitySpatialAwarenessSystem.cs

```diff
@@ -50,6 +54,12 @@ public bool CheckCapability(MixedRealityCapability capability)
 
         private MixedRealitySpatialAwarenessEventData<SpatialAwarenessMeshObject> meshEventData = null;
 
+        /// <inheritdoc />
+        public override void HandleEvent<T>(BaseEventData eventData, ExecuteEvents.EventFunction<T> eventHandler)
+        {
+            base.HandleEvent(eventData, eventHandler);
```

Comment:
What's the purpose of an override that only calls the `base` function?

Reply:
it helped with debugging :D 

i can remove, though we may wish to remove a similar method in other services.

Reply:
removed